packages/mds-utils/utils.ts

```diff
@@ -580,7 +583,7 @@ function isStateTransitionValid(eventA: VehicleEvent, eventB: VehicleEvent) {
   }
 }
 
-function getPolygon(geographies: Geography[], geography: string): Geometry | FeatureCollection {
+function getPolygon(geographies: Geography[], geography: string): Geometry | FeatureCollection | undefined {
   const res = geographies.find((location: Geography) => {
```

Comment:
Are we sure we want an `undefined` and we don't want to throw an error instead?

Reply:
Discussed over slack - I'm fine either way, especially if current conventions here are to throw an error. But I think if we want `Geography.geography_json` to be optional, it makes a little more sense to return `undefined` here right?

Reply:
Nevermind sounds like michael wants it to be undefined!

Reply:
@BigIdeas fair enough